mtags-shared/src/main/scala/scala/meta/internal/metals/ReportContext.scala

```diff
@@ -146,12 +149,18 @@ class StdReporter(
             duplicate <- reportedMap.get(id)
           } yield duplicate
 
-        optDuplicate.orElse {
+        val pathToReport = optDuplicate.getOrElse {
           path.createDirectories()
           path.writeText(sanitize(report.fullText(withIdAndSummary = true)))
-          Some(path)
+          path
+        }
+        if (!ifVerbose) {
```

Comment:
Should we not report it always??

Reply:
In `verbose` reports we have e.g. missing hover, which happens quite often and most of the time it is not a bug. Feels like that might produce too much noise.